worldedit-core/src/main/java/com/sk89q/worldedit/math/BlockVector2.java

```diff
@@ -60,18 +60,6 @@ public static BlockVector2 at(double x, double z) {
     }
 
     public static BlockVector2 at(int x, int z) {
-        switch (x) {
```

Comment:
This & the other switch removals should be reverted, performance tests show that this isn't any more efficient. The change also increases memory usage, making this a net negative change.